<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewConfigTab</class>
  <widget class="QWidget" name="tabViews">
   <attribute name="title">
    <string>Ansichten</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QFrame" name="frameViewsSettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frameViewsSettingsName">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="labelViewsSettingsNameLabel">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Ansicht:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditViewsSettingsNameValue"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameViewsSettingsYAxis">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutViewsSettingsYAxisFirst">
            <item>
             <widget class="QFrame" name="frameViewsSettingsYAxisFirstControls">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>150</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QFormLayout" name="formLayout_4">
               <item row="4" column="0">
                <widget class="QLabel" name="labelViewsSettingsYAxisFirstControlsMeasurementSize">
                 <property name="text">
                  <string>Messgrösse:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="comboBoxViewsSettingsYAxisFirstControlsMeasurementSize">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="labelViewsSettingsYAxisFirstControlsAxisLabel">
                 <property name="text">
                  <string>Bezeichnung:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QLineEdit" name="lineEditViewsSettingsYAxisFirstControlsAxisLabel"/>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="checkBoxViewsSettingsYAxisFirstControlsActive">
                 <property name="text">
                  <string>Aktiv</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="labelViewsSettingsYAxisFirstControlsActive">
                 <property name="text">
                  <string>Linke Y-Achse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="listViewViewsSettingsYAxisFirstSensors">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutViewsSettingsYAxisSecond">
            <item>
             <widget class="QFrame" name="frameViewsSettingsYAxisSecondControls">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>150</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QFormLayout" name="formLayout_3">
               <item row="4" column="0">
                <widget class="QLabel" name="labelViewsSettingsYAxisSecondControlsMeasurementSize">
                 <property name="text">
                  <string>Messgrösse:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="comboBoxViewsSettingsYAxisSecondControlsMeasurementSize">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="labelViewsSettingsYAxisSecondControlsAxisLabel">
                 <property name="text">
                  <string>Bezeichnung:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QLineEdit" name="lineEditViewsSettingsYAxisSecondControlsAxisLabel"/>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="checkBoxViewsSettingsYAxisSecondControlsActive">
                 <property name="text">
                  <string>Aktiv</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="labelViewsSettingsYAxisSecondControlsActive">
                 <property name="text">
                  <string>Rechte Y-Achse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="listViewViewsSettingsYAxisSecondSensors">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonViewsSettingsSave">
            <property name="text">
             <string>Speichern</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameViewsOverview">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Ansichten:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetViewsOverviewControlsViews"/>
       </item>
       <item>
        <widget class="QFrame" name="frameViewsOverviewControls">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButtonViewsOverviewControlsOpen">
            <property name="text">
             <string>Anzeigen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonViewsOverviewControlsNew">
            <property name="text">
             <string>Neu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonViewsOverviewControlsDelete">
            <property name="text">
             <string>Löschen</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 <resources/>
 <connections/>
</ui>
